src/tools/sfdk/command.cpp

```diff
@@ -325,6 +326,26 @@ class VirtualMachinePropertiesAccessor : public PropertiesAccessor
                 return Failed;
             }
             return Prepared;
+        } else if (name == VM_FREE_STORAGE_SIZE_MB) {
+            int commaIndex = value.indexOf(',');
+            QString freeSize = value.left(commaIndex);
+            if (freeSize.endsWith('+')) {
+                freeSize.chop(1);
+            } else {
+                *errorString = valueCannotBeDecreasedMessage();
+                return Failed;
+            }
+            if (!parsePositiveInt(&m_freeSizeMb, freeSize, errorString))
+                return Failed;
+            if (commaIndex > 0) {
+                if (!parsePositiveInt(&m_incrementMb, value.mid(commaIndex + 1), errorString))
+                    return Failed;
+                if (m_incrementMb < m_freeSizeMb) {
+                    *errorString = tr("Increment can't be smaller than minimum free storage size");
```

Comment:
Maybe `<step> cannot be smaller than <size>`, provided that the accepted argument would be documented as `<size>[+][,<step>]`. BTW, the docs need to be updated in the module definition.  (There was also a rule that contractions shouldn't be used in messages IIRC, hence "can't" -> "cannot".)

Reply:
Previously we discussed that this wouldn't be documented, at least for the time being. This was supposed to be used by the installer only.